stingray/simulator/simulator.py

```diff
@@ -28,39 +30,46 @@ class Simulator(object):
     rms : float, default 1
         fractional rms of the simulated light curve,
         actual rms is calculated by mean*rms
+    err : float, default 0
+        the errorbars on the final light curve bins
```

Comment:
This is a bit confusing, because "light curve bins" sounds to me like the errorbars on the x-axis. I don't think that's what you mean?

Reply:
Right, no, I meant the y axis. Good catch